app/components/metadata-status-cell/component.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@glimmer/component';
```

Comment:
Nice!!! I would be in favor of writing all the components in here as Glimmer. Definitely open to chatting about it though.

Reply:
So far, I have found only one case where `@glimmer/component` caused an issue. When creating a custom component to be used with `ember-models-table`, Glimmer components won't have some `ember-models-table` functions/properties attached. This only matters if you need to access specific functions like those that tie into the table rows. `@ember/component` works for these, though. Maybe this is because the `ember-models-table` library needs to be updated?

Reply:
Okay, sounds good. Maybe they'll update it relatively soon here.